### ISSUE TITLE
Wrong mdl version on latest Docker image

### ISSUE THREAD

**Issue by GWillmann** [markdownlint/markdownlint#479]
## Description
Wrong mdl version on latest Docker image

### Environment
On the latest Docker image (`0.13.0` or `latest` tags).

### Expected Behavior
The following command:
```bash
docker run --rm markdownlint/markdownlint --version
```
should return `0.13.0`

### Actual Behavior
It returns `0.12.0`
<img width="823" alt="Screenshot 2023-10-30 at 10 00 01" src="https://github.com/markdownlint/markdownlint/assets/5655900/51e7d766-a7f1-487d-8060-44bc96466c7a">

## Replication Case
```bash
docker pull markdownlint/markdownlint
docker run --rm markdownlint/markdownlint --version
```

**Comment by GWillmann** [comment]
Fixed by releasing https://github.com/markdownlint/markdownlint/releases/tag/v0.15.0 thanks!
